docs/un-supported-patch-properties.md

```diff
@@ -14,7 +14,7 @@ ARM OpenAPI(swagger) specs
 
 ## Output Message
 
-The patch operation body parameter schema should not contains property {propertyName}.
+The patch operation body parameter schema should not contain top level "id", "name", "type", "location" as writable properties. This is because these properties are expected to be either readOnly or immutable.
```

Comment:
```suggestion
The patch operation body parameter schema should not contain top-level "id", "name", "type", "location" as writable properties. To fix this, either remove the offending properties, or mark them as readOnly or immutable.
```

I think this would convey what is going a bit more naturally, based on our discussion.

Reply:
Could also put it in quotes

Reply:
I felt the "How to ix the violation was where we tell what to do to fix the violation so left the Output message to only reflect an explanation of the problem

Reply:
But I will change it anyway.

Reply:
Which part in quotes Brett?

Reply:
I meant to put that reply on https://github.com/Azure/azure-openapi-validator/pull/513#discussion_r1196076146